rust/src/sip/log.rs

```diff
@@ -27,6 +28,12 @@ fn log(tx: &SIPTransaction, js: &mut JsonBuilder) -> Result<(), JsonError> {
         js.set_string("method", &req.method)?
             .set_string("uri", &req.path)?
             .set_string("version", &req.version)?;
+
+        if let Some(sdp_body) = &req.body {
+            js.open_object("sdp")?;
```

Comment:
Wonder if this should be part of `sdp_log` ...

Reply:
`sdp` seems more consistent with other loggers to me, given that each json object is named after the protocol.

@victorjulien @jasonish thoughts?

Reply:
Yes, I meant the line ` js.open_object("sdp")?` should be in the function `sdp_log` rather than its caller

Reply:
Ah, sorry.  I misunderstood.

Reply:
My bad, that was not clear (and I am not sure what is best between caller and callee here)